drivers/video/video_framebuff.c

```diff
@@ -46,24 +46,13 @@ static void init_buf_chain(video_framebuff_t *fbuf)
   fbuf->vbuf_tail  = NULL;
 
   tmp = fbuf->vbuf_alloced;
-  for (i = 0; i < fbuf->container_size - 1; i++)
+  for (i = 1; i < fbuf->container_size; i++)
     {
       tmp->next = &tmp[1];
       tmp++;
     }
 }
 
-static void cleanup_container(video_framebuff_t *fbuf)
-{
-  if (fbuf->vbuf_alloced)
-    {
-      memset(fbuf->vbuf_alloced,
-             0,
-             sizeof(vbuf_container_t)*fbuf->container_size);
-      init_buf_chain(fbuf);
-    }
-}
-
 static inline int is_last_one(video_framebuff_t *fbuf)
```

Comment:
```suggestion
static inline int is_last_one(FAR video_framebuff_t *fbuf)
```

Reply:
@pkarashchenko since the whole video_framebuff.* doesn't add FAR, it's better to do FAR addition to the new patch.